js/app.js

```diff
@@ -81,8 +82,8 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
     var totalProgress = 100;
     $scope.copyshow = false;
     //blockonomics_time_period is defined on JS file as global var
-    var totalTime = blockonomics_time_period * 60;
-    $scope.display_problems = true;
+    var totalTime = 600;
```

Comment:
In order to remove hard-coding the total time. Pass it with through with the order - `$scope.order.time.remaining` and `$scope.order.time.total`.